pipelines/broad/dna_seq/germline/single_sample/exome/ExomeGermlineSingleSample.wdl

```diff
@@ -6,6 +6,10 @@ version 1.0
 ## generation) according to the GATK Best Practices (June 2016) for germline SNP and
 ## Indel discovery in human exome sequencing data.
 ##
+## NOTE: "Reblocking" of the output GVCF, which reduces hom-ref genotype quality resolution
```

Comment:
Is there a comma missing after `resolution`?

And perhaps a bit of tweaking: `"Reblocking" of the output GVCF is now done by default. This reduces the genomic resolution of hom-ref genotype qualities, which may affect de novo calling in trios.` Or something along those lines. (Note also "genomic resolution," which may prevent confusion with decreased resolution of GQ bins or other tangentially related concepts.)